src/prospectScraper/ProspectRanking.cs

```diff
@@ -7,29 +7,16 @@ public class ProspectRanking
     {
         public int Rank { get; set; }
         public string Change { get; set; }
+        public DateTime Date { get; set; }
```

Comment:
I'm not sure about this.  The original reason that I stored the Date as a string is because I never really wanted to get any more specific than a day, and I know DateTime stores a bunch more specific than that.